### PR TITLE
Bug 1566438, added suggestion to validate storage before upgrading

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1566438

@sdodson PTAL. Thanks!

### CODE REVIEW

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -49,6 +49,14 @@ Containerized GlusterFS] before proceeding.
 
 * If you are using GCE Persistent Disk (gcePD), see xref:special-considerations-for-gcepd[Special Considerations When Using
 gcePD] before proceeding.
+
+* The day before the upgrade, validate {product-title} 3.7 storage migration to
+ensure potential issues are resolved prior to the outage window:
++
+----
+$ oc adm migrate storage --include=* --loglevel=2
```
Comment (sdodson) [inline upgrading/automated_upgrades.adoc]: In OCP < 3.10 they need to add `--confirm` in all cases.

**Review by sdodson — CHANGES_REQUESTED** [review]
needs --confirm flag in < 3.10

Also, this pre-upgrade check is a good idea to do in all upgrades, not just limited to 3.7, lets add it to 3.9 and 3.10 too.

**Comment by ahardin-rh** [comment]
@sdodson I updated this PR and applied the changes to the other branches. 
3.10+: qqqhttps://github.com/openshift/openshift-docs/pull/11505
3.9: https://github.com/openshift/openshift-docs/pull/11509

**Comment by sdodson** [comment]
/lgtm 
thanks!

**Comment by ahardin-rh** [comment]
QE verified also

**Pull request merged**